test/xpu_api/random/interface_tests/engines_methods.pass.cpp

```diff
@@ -99,8 +99,6 @@ test_vec
 public:
     bool run(sycl::queue& queue)
     {
-        using result_type = typename Engine::scalar_type;
```

Comment:
I think this was done by the reason
The main aim here - to test that result_type is available in the engine

Reply:
I see. So, we can keep it and make additional changes to avoid the warning about unused variable

Reply:
Sounds good, Andrey